contributors/LesCrous.md

```diff
@@ -0,0 +1,13 @@
+2016-01-06
+
+I hereby agree to the terms of the Contributors License
+Agreement, version 1.0, with MD5 checksum
+2141746be10ee23794f402369cf644fc.
+
+I furthermore declare that I am authorized and able to make this
+agreement and sign this declaration.
+
+Signed,
+
+Olivier Perrot
+https://github.com/LesCrous
```

Comment:
Sorry if the instructions were not clear - but this file needs to go in the xibosignage/xibo repository and **not** in this one.